### PR TITLE
build: update tokenlists test

### PR DESCRIPTION
**Pull request by janndriessen**
* Updates AnalyticsSDK to use new tokenlists
* Temporarily removes official tokenlists package (to be readded when we switched to new version officially)

### CODE REVIEW

File: src/providers/nav/fli-nav-provider.test.ts [hunk 1]
```diff
@@ -9,14 +9,14 @@ const rpcProvider = buildAlchemyProvider(1, process.env.ALCHEMY_API_KEY!)
 
 describe("FliNavProvider", () => {
   test("returns NAV for BTC2x-FLI", async () => {
-    const btc2xfli = getIndexTokenData("BTC2x-FLI")!.address
+    const btc2xfli = tokenSymbolMap[1]["BTC2x-FLI"].address
```
Comment (tokdaniel) [inline src/providers/nav/fli-nav-provider.test.ts:12]: here you can also just do `getTokenByChainAndSymbol(1, 'BTC2xFLI').address`

File: src/providers/nav/fli-nav-provider.test.ts [hunk 2]
```diff
@@ -9,14 +9,14 @@ const rpcProvider = buildAlchemyProvider(1, process.env.ALCHEMY_API_KEY!)
...
     const provider = new FliNavProvider(rpcProvider, coingeckoService)
     const nav = await provider.getNav(btc2xfli)
     expect(nav).toBeGreaterThan(1)
   })
 
   test("returns NAV for ETH2xFLI", async () => {
-    const eth2xfli = getIndexTokenData("ETH2x-FLI")!.address
+    const eth2xfli = tokenSymbolMap[1]["ETH2x-FLI"].address
```
Comment (tokdaniel) [inline src/providers/nav/fli-nav-provider.test.ts:19]: same

File: src/providers/analytics/provider.ts [hunk 3]
```diff
@@ -58,7 +58,10 @@ export class IndexAnalyticsProvider implements AnalyticsProvider {
       include24hrVolume: true,
     },
   ): Promise<IndexAnalytics> {
-    const token = getIndexTokenDataByAddress(address, chainId)
+    const indices = getChainTokenList(chainId, ["index"])
+    const token = indices.find((token) =>
```
Comment (tokdaniel) [inline src/providers/analytics/provider.ts:62]: also here if you need an individual token, just do

```typescript
const token = getTokenByChainIdAndAddress(chainId, address)
```

Comment (janndriessen) [reply src/providers/analytics/provider.ts:62]: We want to make sure to only have our indices here

Comment (tokdaniel) [reply src/providers/analytics/provider.ts:62]: ah i see okay

**Review by tokdaniel — APPROVED** [review]
image

File: src/providers/nav/fli-nav-provider.test.ts [hunk 4]
```diff
@@ -9,14 +9,14 @@ const rpcProvider = buildAlchemyProvider(1, process.env.ALCHEMY_API_KEY!)
 
 describe("FliNavProvider", () => {
   test("returns NAV for BTC2x-FLI", async () => {
-    const btc2xfli = getIndexTokenData("BTC2x-FLI")!.address
+    const btc2xfli = getTokenByChainAndSymbol(1, "BTC2x-FLI").address
```
Comment (0xonramp) [inline src/providers/nav/fli-nav-provider.test.ts:12]: cool to see the typesafety on the second arg 🔥

**Review by 0xonramp — APPROVED** [review]
LGTM

**Pull request merged**